File: URI/concluidos/Javascript/1010.js
var input = require('fs').readFileSync('./dev/stdin/arquivo.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (line) => line.split(" ").map(a => +a);
const [a, b, c] = pegarValores(lines.shift());
const [d, e, f] = pegarValores(lines.shift());

console.log(`VALOR A PAGAR: R$ ${((b*c) + (e*f)).toFixed(2)}`);

File: URI/concluidos/Python/1010.py
linha = input().split(" ")
code, npecas, vlunit = linha
linha = input().split(" ")
code1, npecas1, vlunit1 = linha

print('VALOR A PAGAR: R$ %0.2f' % ((int(npecas) * float(vlunit)) + (int(npecas1) * float(vlunit1))))